internal/node/validate.go

```diff
@@ -140,7 +153,19 @@ func (a APIServerValidator) CheckVPCEndpointAccess(ctx context.Context, informer
 		return err
 	}
 
-	kubeEndpoint, err := client.CoreV1().Endpoints("default").Get(ctx, "kubernetes", metav1.GetOptions{})
+	var kubeEndpoint *v1.Endpoints
+	consecutiveErrors := 0
+	err = wait.PollUntilContextTimeout(ctx, validation.ValidationInterval, validation.ValidationTimeout, true, func(ctx context.Context) (bool, error) {
```

Comment:
are these all the requests we make from debug? i suspect there are more. at least we have one that calls the selfsubjectreview api, there might be more